Xamarin.Forms.Platform.Android/CollectionView/ItemsViewAdapter.cs

```diff
@@ -66,6 +65,22 @@ public override RecyclerView.ViewHolder OnCreateViewHolder(ViewGroup parent, int
 			return new TemplatedItemViewHolder(itemContentControl, templateElement);
 		}
 
+		protected override void Dispose(bool disposing)
+		{
+			if (!_disposed)
+			{
+				if (disposing)
+				{
+					_itemsSource?.Dispose();
+					_itemsSource = null;
```

Comment:
Setting `_itemsSource` and `_createView` to null doesn't do anything here, these can be removed.